modules/logrotate/templates/logrotate.conf.erb

```diff
@@ -41,4 +44,10 @@
     # will refuse to rotate the logs if an empty .1 log file exists.
     find <%= @matches %>* -wholename '/var/log/*' -name '*.1' -size 0 -delete
   endscript
+  lastaction
+    # Clean up files older than maximum rotation. No-op unless "rotate" value is reduced,
+    # because logrotate only deletes rotate+1, not all older files.
+    # Thanks to https://serverfault.com/questions/636844/making-logrotate-remove-old-logs-after-reducing-rotate-value
+    find <%= @matches %>.*  -maxdepth 1 -mtime +<%= @days_to_keep %> -delete &>/dev/null
```

Comment:
Rather than redirecting stdout to null I think it'd be slightly nicer if we had stdout that said what this was doing. From a quick test:

```
find <%= @matches %>.* -maxdepth 1 -mtime +<%= @days_to_keep %> -delete -printf "deleting old file: %p\n"
```

should add to the output e.g:

```
...
old log /var/log/content-store/app.out.log.0.gz does not exist
copying /var/log/content-store/app.out.log to /var/log/content-store/app.out.log.1
truncating /var/log/content-store/app.out.log
compressing log with: /bin/gzip
removing old log /var/log/content-store/app.out.log.8.gz
running last action script
deleting old file: /var/log/content-store/app.err.log.9.gz
deleting old file: /var/log/content-store/app.out.log.9.gz
```

Reply:
Great point @kevindew, I've updated things to run exactly that. 🙇